src/DevExpressApp/MainWindow.xaml

```diff
@@ -3,30 +3,97 @@
+                <Setter.Value>
+                    <ControlTemplate TargetType="ToggleButton">
+                        <Border x:Name="border"
+                                Background="{TemplateBinding Background}"
+                                CornerRadius="0"
+                                BorderBrush="Transparent"
+                                BorderThickness="1"
+                                Padding="0">
+                            <ContentPresenter HorizontalAlignment="Center" VerticalAlignment="Center"/>
+                        </Border>
+                        <ControlTemplate.Triggers>
+                            <Trigger Property="IsMouseOver" Value="True">
+                                <Setter TargetName="border" Property="Background" Value="#F28C28"/>
+                                <Setter Property="Foreground" Value="White"/>
+                            </Trigger>
+                            <Trigger Property="IsPressed" Value="True">
+                                <Setter TargetName="border" Property="Background" Value="#E67E22"/>
+                            </Trigger>
+                            <Trigger Property="IsChecked" Value="True">
+                                <Setter TargetName="border" Property="Background" Value="White"/>
+                                <Setter Property="Foreground" Value="#F28C28"/>
+                            </Trigger>
+                        </ControlTemplate.Triggers>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+    </Window.Resources>
 
-            <!-- Time Picker -->
-            <TabItem Header="Time Picker">
-                <local:TimePickerView />
-            </TabItem>
+    <Grid Margin="10">
+        <DockPanel LastChildFill="True">
+            
+            <Border DockPanel.Dock="Left"
+                    Width="180"
+                    Background="#F8F8F8"
+                    CornerRadius="6"
+                    BorderBrush="#D0D0D0"
+                    BorderThickness="1"
+                    Margin="0 0 15 0"
+                    Padding="10">
 
-            <!-- Toggle Switch -->
-            <TabItem Header="Toggle Switch">
-                <local:ToggleSwitchView />
-            </TabItem>
-
-            <!-- Track Bar -->
-            <TabItem Header="Track Bar">
-                <local:TrackBarView />
-            </TabItem>
-        </TabControl>
+                <DockPanel LastChildFill="True">
+                    <Image Source="Assets/dev-express.png"
+                           Height="110"
+                           Width="500"
+                           Stretch="Uniform"
+                           DockPanel.Dock="Top"
+                           Margin="0 0 0 20"
+                           HorizontalAlignment="Center"/>
+                    
+                    <StackPanel>
+                        <ToggleButton x:Name="BtnBarCode" Content="Bar Code"
```

Comment:
Nit: given that you're restyling a control here (`ToggleButton`), it might actually make more sense to use `RadioButton`s as these will automatically ensure that only one is selected.

**However**, that's not the exercise here so `ToggleButton`s are is perfectly fine for now ;) Just for future reference if you'd like to investigate that.